services/gtfs-ckan-uploader/calitp_data_ca_exporter.py

```diff
@@ -0,0 +1,90 @@
+import os
+import requests
+import tempfile
+from calitp import get_table
+
+
+#
+API_ENDPOINT = "https://data.ca.gov/api/3/action/resource_update"
+API_KEY = os.environ.get("CALITP_CKAN_GTFS_SCHEDULE_KEY")
+
+#
+
+with tempfile.TemporaryDirectory() as tmp_dir:
+    print("Downloading agency")
+    agency = get_table("gtfs_schedule.agency", as_df=True)
+    agency.to_csv(f"{tmp_dir}/gtfs_schedule_agency.csv")
+
+    print("Posting agency")
+    r = requests.post(
+        API_ENDPOINT,
+        data={"id": "e8f9d49e-2bb6-400b-b01f-28bc2e0e7df2"},
+        headers={"Authorization": API_KEY},
+        files={"upload": "gtfs_schedule_agency.csv"},
```

Comment:
This isn't the file name, so it isn't actually uploading the file to the dataportal. needs to be done with `{tmp_dir}/path`. So they dataset

Reply:
cc @Nkdiaz

Reply:
We need to replace the string (e.g. "gtfs_schedule_agency.csv") with the contents of the file, (e.g. `open(FILE_NAME, "rb")`)

https://stackoverflow.com/a/52396494/1144523

Reply:
```
files={"upload":  open(f"{tmp_dir}/gtfs_schedule_agency.csv", "rb")}
```